packages/devextreme/testing/testcafe/helpers/accessibility/test.ts

```diff
@@ -19,17 +19,17 @@ const defaultA11yCheckConfig = isMaterialBased() ? {
   runOnly: 'color-contrast',
 } : {};
 
-const getOptionConfigurations = (options: Options | undefined) => {
+const getOptionConfigurations = <TComponentOptions = unknown>(
+  options: Options<TComponentOptions> | undefined,
+): TComponentOptions[] => {
   if (!(options && Object.keys(options).length)) {
-    return [defaultOptions];
+    return [defaultOptions as TComponentOptions];
```

Comment:
Minor comment.

Can we return just an empty array here instead of the array with one item (empty object)?
```suggestion
    return [];
```